internal/genenv/certs.go

```
@@ -0,0 +1,53 @@
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+func GenerateSelfSignedCerts(base string) error {
+	certs := []string{"mtls.localhost", "admin.localhost"}
+	certDir := filepath.Join(base, "certs")
+	os.MkdirAll(certDir, 0755)
```

golangci-lint (1.64.8)

Error return value of `os.MkdirAll` is not checked

(errcheck)